hardware/hardware.mk

```diff
@@ -157,7 +157,7 @@ $(BSG_MACHINE_PATH)/bsg_bladerunner_configuration.rom: $(BSG_MACHINE_PATH)/Makef
 	@echo $(call dec2bin,$(BSG_MACHINE_HOST_COORD_Y))      >> $@.temp
 	@echo $(call dec2bin,$(BSG_MACHINE_NOC_COORD_X_WIDTH)) >> $@.temp
 	@echo $(call dec2bin,$(BSG_MACHINE_NOC_COORD_Y_WIDTH)) >> $@.temp
-	@echo $(call dec2bin,0)                                >> $@.temp
+	@echo $(call dec2bin,$(BSG_MACHINE_RUCHE_FACTOR_X))    >> $@.temp
```

Comment:
Did you want to print barrier ruche factor or network ruche factor?

Reply:
I realize that all the code reads the machine ruche factor, but doesn't the barrier use the barrier ruche factor?

Reply:
Well flip that question around and ask why there are two different parameters? Can the barrier ruche factor be different from the NOC ruche factor?

Reply:
@tommydcjung